pennylane/transforms/core/transform_program.py

```diff
@@ -284,6 +284,7 @@ def add_transform(self, transform: TransformDispatcher, *targs, **tkwargs):
                 targs,
                 tkwargs,
                 transform.classical_cotransform,
+                transform.plxpr_transform,
                 transform.is_informative,
                 transform.final_transform,
```

Comment:
Can we start passing these as keyword arguments?
```suggestion
                classical_cotransfrom=transform.classical_cotransform,
               plxpr_transform= transform.plxpr_transform,
                is_informative=transform.is_informative,
                final_transform=transform.final_transform,
```